esp/templates/contact.html

```diff
@@ -26,13 +28,14 @@
   {% inline_qsd_block "contact_header_text" %}
   # Contact Us
 
-  Fill out this form to send a quick e-mail to the {{ settings.ORGANIZATION_SHORT_NAME }} office. We will get back to you as soon as possible (usually within a day). Submitting this form does not add you to our mailing list.
-
-  <small>(Unless of course, you, uh, ASK to be added to our mailing list.)</small>
```

Comment:
Awwwwww :(

Reply:
Sorry :/ I feel like it might not be the *best* default (?)

Reply:
Especially since the mailing list feature is defunct (or so I believe)

Reply:
I don't know, I always thought it was a kinda fun silly thing.  Keeps us human, you know.  But if you want to get rid of it that's ok.